Sources/SpeziViews/Views/Text/MarkdownView.swift

```diff
@@ -95,14 +96,18 @@ public struct MarkdownView: View {
     /// Parses the incoming markdown and handles the view's error state management.
     /// - Parameters:
     ///   - markdown: A `Data` instance containing the markdown file in an utf8 representation.
-    ///
+    ///   ```swift
+    ///   // Example
+    ///   Data(" # This is a markdown example
+    ///    *This should be italiced* and **this bolded**").utf8
+    ///    ```
```

Comment:
This code example does not compile, if you want to create a String with line breaks you will have to use a multi-line string interval: https://docs.swift.org/swift-book/documentation/the-swift-programming-language/stringsandcharacters/#Multiline-String-Literals

In addition, the code example does not call the API.

Comments like `//Example` should be removed and I would suggest to rather add an explanation around the example block.

Last but not least, I would move the example above the parameters section of the code documentation.